identity/client/e2e/pages/UsersPage.ts

```diff
@@ -7,8 +7,11 @@
  */
 
 import { Page, Locator } from "@playwright/test";
+import { Paths } from "../utils/paths";
+import { relativizePath } from "../utils/relativizePaths";
```

Comment:
It seems that we keep calling `relativizePath` in all paths.
Would it be possible to get rid of this function and make all the paths relative.

Reply:
There are some usages of paths that are not relative in the login tests, for example. Going forward we will need the paths for any api URLs we might have to await for tests, as well as any nested urls (i.e. /users/<id>/<groupsPath>) and if the paths have the dot on them they don't work for any composite usage. 

Given those points, I believe we should maintain this usage, as over time we will most likely have more usage of paths without being relative. We do the same (with the same function) for the Operate tests, for example.

Reply:
I see your point. However, it is just that we have to keep calling "relativizePath" everywhere, and i am thinking there is probably a better way of doing this. No need to adjust this for now. I will look into this later and see if we can  improve this somehow.